test/fixtures/vcr/relay_group_add_relay.json

```diff
@@ -11,14 +11,14 @@
       "url": "http://localhost:4000/v1/token"
     },
     "response": {
-      "body": "{\"token\":{\"value\":\"w05n5iZL4peQoZ1gibP1MR1TgvSTXrIwZzCIJYRjR+0=\"}}",
```

Comment:
It doesn't look like anything significant actually changed in this fixture, and I think a few others are like that, too. If that is indeed the case, it'd be nice to exclude these changes from the PR, just to keep it lean and focused.